subsys/bluetooth/controller/ll_sw/nordic/lll/lll_sync.c

```diff
@@ -548,8 +556,12 @@ static void isr_rx_adv_sync(void *param)
 	/* Save radio ready and address capture timestamp for later use for
 	 * drift compensation.
 	 */
-	radio_tmr_aa_save(radio_tmr_aa_get());
-	radio_tmr_ready_save(radio_tmr_ready_get());
+	if (err != -EINVAL) {
+		trx_cnt = 1U;
```

Comment:
Question to get correct understanding. The timers required to evaluate drift compensations are saved every correctly received PDU, including chained PDUs. The `isr_rx_done_cleanup` is called after last chained PDU is received, then drift compensation data are forwarded to ULL by EVENT_DONE_EXTRA_TYPE_SYNC. 
Am I correct?

Do you see any problem if that is updated when first per. sync. train PDU is received?

Reply:
Ah... @carlescufi was fast enough to merge it...

anyway @cvinayak answer for my comment please. It would help me to rebase my PR https://github.com/zephyrproject-rtos/zephyr/pull/38292

Reply:
The timers required to evaluate drift compensations are saved only for the AUX_SYNC_IND PDU (irrespective of CRC valid or invalid), which is the anchor point for the radio events. Chain PDUs should not change the saved timestamp, they are not anchor points for the periodic advertising radio events. The saved timestamps are used in the ULL when processing done event.